File: 4. Hashing/count.py
#Write a function to count frequency of elements in an array.
arr = [1,2,1,3,3,4,4,5,2,2]
map = {}

def countFrequency(arr):
    for item in arr:
        if item not in map:
            map[item] = 1
        else:
            map[item] += 1
    print(map)
    return map

countFrequency(arr)

File: 2. Hashing/count.js
//Count the frequency of occurance of elements of an array.
let arr = [1,2,1,3,3,4,4,5,2,2,2]

function countFrequency(arr){
    let count = 0;
    let map = new Map();
    for(let i = 0; i < arr.length; i++){
        if(!map.has(arr[i])){
            map.set(arr[i], count+1);
        }else{
            map.set(arr[i], map.get(arr[i])+1);
        }
    }
    console.log(map);
    return map;
}

countFrequency(arr)

